src/theme/theme.ts

```diff
@@ -127,7 +127,7 @@ declare module "@mui/material/Typography" {
 }
 
 const colors = {
-  blue: "#0099CC",
+  blue: "#3985F7",
```

Comment:
Changed `theme.colors` blue to match color blue provided in [info alert figma design ](https://www.figma.com/file/Z1yL2dk96S3CBB4CdnUf9E?node-id=1264:1041#372553992) requested by Tizi, 
The blue isn't used anywhere else besides the wallet page, which isn't being loaded anyway. 

Noticed colors haven't been updated to current artist-portal theme, but feel like that lift will only be needed if we end up doing more color changes/styles.